core-skills/react/functional-and-class.md

```diff
@@ -0,0 +1,10 @@
+# React - Functional and Class Components
+
+An Engineer with this core skill should be able to write components as either class based or functional components
+and understand what is and isn't possible with each type of component.
```

Comment:
As I understand it you can effectively do the same things with both functional and class components now, it's just that certain things may be less practical with each approach, what do you think of:
```suggestion
and decide what is best suited for their use case.
```

Reply:
Yeah, that makes sense to me 👍